akka-actor-typed/src/main/resources/reference.conf

```diff
@@ -127,3 +127,16 @@ akka.reliable-delivery {
     }
   }
 }
+
+
+akka.actor {
+  serializers {
+    akka-actor-typed-entity = "akka.actor.typed.internal.entity.EntityEnvelopeSerializer"
+  }
+  serialization-bindings {
+    "akka.actor.typed.EntityEnvelope" = akka-actor-typed-entity
+  }
+  serialization-identifiers {
+    "akka.actor.typed.internal.entity.EntityEnvelopeSerializer" = 41
```

Comment:
I saw that we just added 39, so I tried 40, but it's already picked. 

What are the rules for this? And how can have an overview of all identifiers in use?

Reply:
they are defined in reference.conf in section `serialization-identifiers`, so the way is to search for a free spot

by the way, 41 is taken by PingSerializer in some test, but we can change that to a higher number

feels wrong to have serialization in this local module, though. It can be defined in Cluster Sharding (maybe in some of the existing serializers)

Reply:
Yes, I was not so happy to have it here, but I think we can remove it all together.

Reply:
@patriknw, it's not possible to remove the serialization. It's  possible to workaround it when using the `EntityRef`, but not when using the shard region `ActorRef` (the one returned by `initEntity`).  Whatever we pass here can end up on the wire before we can do any conversion. 

So, I kept the serialization, but now moved to cluster sharding. It's a little odd, because it's a class defined in `akka-actor-typed`, but the serialization is `akka-cluster-sharding-typed`. Anyway, I don't see any other use case where the serialization will be needed while not using sharding.

I think it's fair to put a limit on it and say that the new entity, if used over remote, must be in the context of cluster sharding. In other words, not remote entity if not sharded.